Fri Mar 17 20:38:02 CET 2006  VMiklos <vmiklos@frugalware.org>
  * _alpm_add_commit(): fix changeleg file's location
diff -rN -u old-pacman/lib/libalpm/add.c new-pacman/lib/libalpm/add.c
--- old-pacman/lib/libalpm/add.c	2006-03-17 20:42:49.000000000 +0100
+++ new-pacman/lib/libalpm/add.c	2006-03-17 20:42:49.000000000 +0100
@@ -441,8 +441,8 @@
 					!strcmp(pathname, ".CHANGELOG")) {
 					if(!strcmp(pathname, ".CHANGELOG")) {
 						/* the changelog goes inside the db */
-						snprintf(expath, PATH_MAX, "%s/%s/%s-%s/changelog", db->path,
-							db->treename, info->name, info->version);
+						snprintf(expath, PATH_MAX, "%s/%s-%s/changelog", db->path,
+							info->name, info->version);
 					} else {
 						/* the install script goes inside the db */
 						snprintf(expath, PATH_MAX, "%s/%s/%s-%s/install", db->path,

